lib/node_modules/@stdlib/utils/is-uppercase/lib/is_uppercase.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+// MODULES //
+
+var lowercase = require( '@stdlib/string/lowercase' );
+var uppercase = require( '@stdlib/string/uppercase' );
+var isString = require( '@stdlib/utils/is-string' ).isPrimitive;
+
+
+// MAIN //
+
+/**
+* Tests if a value is an uppercase string.
+*
+* @param {*} value - value to test
+* @returns {boolean} boolean indicating whether value is an uppercase string
+*
+* @example
+* var bool = isUppercase( 'HELLO' );
+* // returns true
+* @example
+* var bool = isUppercase( 'World' );
+* // returns false
+* @example
+* var bool = isUppercase( '!' );
+* // returns false
+* @example
+* var bool = isUppercase( 'salt and light' );
+* // returns false
+*/
+function isUppercase( value ) {
+	if ( isString( value ) ) {
```

Comment:
This can be simplified to

``` javascript
return (
    isString( value )  &&
    value === uppercase( value ) &&
    value !== lowercase( value )
);
```